client/src/app/+accounts/accounts.component.ts

```diff
@@ -70,4 +70,18 @@ export class AccountsComponent implements OnInit, OnDestroy {
           )
     }
   }
+
+  copyText(val: string){
```

Comment:
Declaration of public instance method not allowed after declaration of private instance method.

Reply:
Fixed